application/backend/app/alembic/versions/da385d690aae_schema.py

```diff
@@ -30,6 +30,7 @@ def upgrade() -> None:
         sa.Column("created_at", sa.DateTime(), server_default=sa.text("(CURRENT_TIMESTAMP)"), nullable=False),
         sa.Column("updated_at", sa.DateTime(), server_default=sa.text("(CURRENT_TIMESTAMP)"), nullable=False),
         sa.PrimaryKeyConstraint("id"),
+        sa.Index("idx_projects_name", "name"),
```

Comment:
@itallix do we also need `__table_args__` as in #4948 ?

<img width="751" height="68" alt="image" src="https://github.com/user-attachments/assets/44e21189-56c9-40a5-b92a-32ea79fc149b" />

Reply:
Yes, indexes should be defined in the `__table_args__` of each corresponding model, as the revision file is generated based on it.